apps/mymedicare_cb/authorization.py

```diff
@@ -9,15 +9,18 @@
 class OAuth2Config(object):
     token_endpoint = settings.SLS_TOKEN_ENDPOINT
     redirect_uri = settings.MEDICARE_REDIRECT_URI
+    # IR TODO - we should require SSL verification, at least in prod, rather than defaulting to False
```

Comment:
I've captured some of this work here previously, sitting in backlog. https://jira.cms.gov/browse/BLUEBUTTON-308